assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -165,6 +165,10 @@ che.docker.namespace=NULL
 # Docker unused containers cleanup period
 che.docker.unused_containers_cleanup_min=60
 
+# Docker container to use volumes-from for workspace containers, used to share ws-agent 
+# without using host directory
+che.docker.volumes_from=che-server
```

Comment:
I agree.  We already have 'che.workspace.volume=' in their for the agent mounting of shared folders.  So to be consistnent we should then name this 'che.workspace.voluems_from='.